src/YoutubeAtom.tsx

```diff
@@ -1,10 +1,10 @@
-import React from 'react';
-import { css } from 'emotion';
+import React, { useEffect, useState, useCallback } from 'react';
 
 import { MaintainAspectRatio } from './common/MaintainAspectRatio';
-
 import { YoutubeOverlay } from './YoutubeOverlay';
 
+declare let window: any;
```

Comment:
This seems strange, isn't `window` already a global?

Reply:
Good point, I think the way we do types in this repo isn't ideal. We should not need to export each type. I think however that would be beyond the scope of this PR. I am happy to put it in the `types.ts` file in the meantime? Currently I havent found any other solution to making TS happy with `window.YT`